index.html

```diff
@@ -4,10 +4,106 @@
+        </ul>   
+    <li>Погружение в HTML-разметку</li>
+        <ul>
+            <li>Мета-теги.</li>
+            <li>Мнемоники.</li>
+            <li>Тег для работы с изображениями</li>
+            <li>Теги для работы со списками.</li>
+            <li>Теги для работы с таблицами.</li>
+            <li>Теги для работы с формами.</li>
+            <li>Div-блоки</li>
+        </ul>
+</ol>
+
+<table border="1">
+    
+    <tr>
+        <th>Tag</th>
+        <th>Description</th>
+    </tr>
+    <tr><td>< table ></td><td>Defines a table</td>
+    <tr><td>< tr ></td><td>Drfines a row in a table</td>
+    <tr><td>< th ></td><td rowspan="2">Defines a "header cell" / "cell" in a table</td>
+    <tr><td>< td ></td>
+    <tr><td colspan="2">Rarely used</td>
+    <tr><td>< caption ></td><td>Defines a table caption</td>
+    <tr><td>< thead ></td><td>Groups the header content in a table</td>
+    <tr><td>< tbody ></td><td>Groups the body content in a table</td>
+    <tr><td>< tfoot ></td><td>Groups the footer content in a table</td>
+    
+</table>
+
+<form>
+    <fieldset>
+            <legend>My first form</legend>
+                <p>Name: <input required name="1"></p>
+                <p>Email: <input required type="email" name="1"></p>
+                <p>Password: <input required type="password" name="1"></p>
+
+        <p>Your programming level</p>     
+            <input class="range-style" type="range" min="0" max="10" step="1" name="Lewel" value="0-10"></p>
+            
+        <p>What does HTML stand for?</p>
+            <p><input type="checkbox"  name="1" value="Hypelinks and Text Markup Language" >Hypelinks and Text Markup Language</p>
+            <p><input type="checkbox"  name="1" value="Home Tool Markup Language" >Home Tool Markup Language</p> 
+            <p><input type="checkbox"  name="1" value="Hyper Text Markup Language" >Hyper Text Markup Language</p>
+
+        <p>What does HTTP stand for?</p>
+            <p><input type="radio" name="1" value="Hyper Text Translate Protocol" >Hyper Text Translate Protocol</p>
+            <p><input type="radio" name="1" value="Hyper Text Transmission Protocol" id="">Hyper Text Transmission Protocol</p>
+            <p><input type="radio" name="1" value="Hyper Text Transfer Protocol" >Hyper Text Transfer Protocol</p>
+            
+        <p>Comment</p>
+        <textarea name="1" id="" cols="20" rows="5" maxlength="500" placeholder="Comment"></textarea>
+
+
+        <p>What does HTML stand for?</p>
+        <select name="1" >
+            <option value="Your answer.">Your answer.</option>
+            <option value="Hypelinks and Text Markup Language">Hypelinks and Text Markup Language</option>
+            <option value="Home Tool Markup Language">Home Tool Markup Language</option>
+            <option value="Hyper Text Markup Language">Hyper Text Markup Language</option>
+        </select>
+
+        <p><input type="submit" value="Send"></p>
+
+    </fieldset>
+
+</form>
+
+
+
```

Comment:
Extra spaces are helpful sometimes when we want to separate "sections" from each other.  But it is better not to abuse them and not use more than 1 in a row, please check the file for their redundancy